docs/sql-server/fabric-database/fabric-mirrored-databases.md

```diff
@@ -22,24 +22,24 @@ For more information, see:
 
 This feature is not currently available for Azure SQL Managed Instance or SQL Server instances.
 
-## What are Fabric mirrored databases?
+## What are mirrored databases in Fabric?
 
 Mirroring in Fabric provides an easy experience to speed the time-to-value for insights and decisions, and to break down data silos between technology solutions, without developing expensive Extract, Transform, and Load (ETL) processes to move data.
 
 With the most up-to-date data in a queryable format in OneLake, you can now use all the different services in Fabric, such as running analytics with Spark, executing notebooks, data engineering, visualizing through Power BI Reports, and more.
 
-With Mirroring in Fabric, you don't need to piece together different services from multiple vendors. Instead, you can enjoy a highly integrated, end-to-end, and easy-to-use product that is designed to simplify your analytics needs, and built for openness and collaboration between technology solutions that can read the open-source Delta Lake table format.
+With mirroring, you don't need to piece together different services from multiple vendors. Instead, you can enjoy a highly integrated, end-to-end, and easy-to-use product that is designed to simplify your analytics needs, and built for openness and collaboration between technology solutions that can read the open-source Delta Lake table format.
```

Comment:
We want to make sure this sentence isn't confusing or taken out of context. It's about Fabric, not SQL Server Database Mirroring.  We can bring clarity.